modules/nw-udn-cr.adoc

```diff
@@ -16,6 +16,20 @@ The following procedure creates a user-defined network that is namespace scoped.
 
 .Procedure
 
+. Optional: For a `UserDefinedNetwork` CR that uses a primary network, create a namespace with the `k8s.ovn.org/primary-user-defined-network` label by entering the following command:
```

Comment:
Please replace "type: NetworkReady" with "type: NetworkCreated"

```
apiVersion: k8s.ovn.org/v1
kind: UserDefinedNetwork
metadata:
  creationTimestamp: "2025-02-13T19:33:01Z"
  finalizers:
  - k8s.ovn.org/user-defined-network-protection
  generation: 1
  name: udncrd-layer2-primary-ipv4
  namespace: test-ipv4-layer2
  resourceVersion: "68437"
  uid: 950d8865-bd0a-4f4f-8d9e-a72f67afdd5b
spec:
  layer2:
    mtu: 9000
    role: Primary
    subnets:
    - 20.10.0.0/16
  topology: Layer2
status:
  conditions:
  - lastTransitionTime: "2025-02-13T19:33:01Z"
    message: NetworkAttachmentDefinition has been created
    reason: NetworkAttachmentDefinitionCreated
    status: "True"
    type: NetworkCreated

```

Reply:
Updated @weliang1 Thanks for pointing that out.